### ISSUE TITLE
Google+ Hangouts API closing?

### ISSUE THREAD

**Issue by dmhendricks** [yakyak/hangupsjs#69]
Forgive my ignorance, but does this mean that the [YakYak](https://github.com/yakyak/yakyak) client and/or `hangupjs` will stop working on 25 Apr 2017?

- [Announcement: Google discontinuing API for Hangouts](https://developers.google.com/+/hangouts/support-faq)

Thank you,
Daniel

**Comment by lorenzos** [comment]
I don't think so. From my understanding the one that are discontinued are not the API used in this repo, but the **Google+** Hangouts API, which are currently used to develop the Hangouts "application", the ones you can see for example in the Hangouts (video)call window (like screensharing, photo sharing, YouTube shared playlists, etc).

Rather, I'd be "worried" by this instead: [Google Hangouts is getting a major overhaul to take on Slack - The Verge](http://www.theverge.com/2017/3/9/14864552/google-hangouts-chat-update-announced-slack-group-messaging-video). Pretty sure our reverse-engineered APIs will be changed, and new (this time, official) APIs for bots and clients will be released.

**Comment by averissimo** [comment]
@dmhendricks this topic also appeared in yakyak and in the parent hangupjs 

https://github.com/yakyak/yakyak/issues/541 and https://github.com/tdryer/hangups/issues/293

as @lorenzos said, that announcement should not have any impact, but the other one he referred could. 

It can be a good thing, as moving to a stable public API would be better than using a reversed engineered web api.. _(an open protocol would be the ideal scenario, but google abandoned that pledge long ago with hangouts release)_

but odds are that google would change the protocol rather abruptly which would lead to some down time.
